lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -325,12 +325,7 @@ function NearbyView({
           />
         </InvisibleA11yLabel>
       )}
-      <NearbySidebarContainer
-        className="base-color-bg"
-        style={{ marginBottom: 0 }}
-      >
-        {/* This is used to scroll to top */}
-        <div aria-hidden ref={firstItemRef} />
+      <div style={{ padding: '1em' }}>
```

Comment:
I think this padding is causing the percy tests to fail. Maybe try this and see if they pass?
```suggestion
      <div style={{ padding: '1em 1em 0 1em' }}>
```

Reply:
Great fix, thank you!